sdk/formrecognizer/Azure.AI.FormRecognizer/src/AnalyzeResult.cs

```diff
@@ -12,9 +11,6 @@ namespace Azure.AI.FormRecognizer.DocumentAnalysis
     [CodeGenModel("AnalyzeResult")]
     public partial class AnalyzeResult
     {
-        /// <summary> Detected languages. </summary>
-        internal IReadOnlyList<DocumentLanguage> Languages { get; }
```

Comment:
Deleted this so CodeGen starts generating this property with a `public` access modifier.